### PR TITLE
Upgrade to use Spree 3.3

### PR DESCRIPTION
**Pull request by vzqzac**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,11 @@
 source 'https://rubygems.org'
 
-gem 'spree', github: 'spree/spree', branch: '3-0-stable'
+gem 'spree', github: 'spree/spree', branch: '3-3-stable'
```
Comment (gus4no) [inline Gemfile]: `github: '', branch: ''` format no longer required, use just `gem spree, ~> '3.3'`

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 # Provides basic authentication functionality for testing parts of your engine
-gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: '3-0-stable'
+gem 'spree_auth_devise', '~> 3.3'
+
+group :development, :test do
+  gem 'pry'
```
Comment (gus4no) [inline Gemfile]: this needed?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -29,10 +29,17 @@
 require 'spree/testing_support/controller_requests'
 require 'spree/testing_support/factories'
 require 'spree/testing_support/url_helpers'
+require 'spree/testing_support/preferences'
+
+require 'spree/api/testing_support/helpers'
+require 'spree/api/testing_support/setup'
+require 'pry'
```
Comment (gus4no) [inline spec/spec_helper.rb]: 👀

Comment (vzqzac) [reply spec/spec_helper.rb]: For testing?

**Pull request merged**